benchmarks/infra/stage-1/modules/gke-infra/cluster.tf

```diff
@@ -0,0 +1,249 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
```

Comment:
Naive question, but is it possible to re-use the existing `infrastructure/` modules for cluster creation? I think we want to avoid multiple modules that just create clusters

Reply:
+1 please can we just have 1 module

Reply:
This piece was part of the discussions of larger refactor that happened prior to OSS - the main reason we kept the two modules was that there were large amounts of variables exposed and having an additional separation would simplify the top level interactions. We plan to follow up with flattening the modules if the overhead of maintenance becomes too much.

Thoughts on doing this as a follow up change in the next few days rather than blocking this change on that suggestion?

Reply:
+1 we should converge on the existing infrastructure/ modules

One thing I noticed - infrastructure utilizes the modules from: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine

Whereas our current tf scripts utilize the modules from: https://github.com/GoogleCloudPlatform/cloud-foundation-fabric

Given the underlying modules are different, this seems like a non-trivial change. It might be better to refactor this post-submit since it'll make it easier to validate and safer to add smaller changes to the infrastructure/ modules if there's additional things that we need to add. We already have 100+ files in this change.